wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -233,6 +233,14 @@ def wait_job_launch(
         )
         return pod_names
 
+    def get_namespace(
+        self, resource_args: Dict[str, Any], default_namespace: str
+    ) -> str:
+        runner_config_namespace = self.backend_config.get("runner", {}).get("namespace")
+        if runner_config_namespace is not None:
+            return runner_config_namespace  # type: ignore
+        return resource_args.get("namespace", default_namespace)
```

Comment:
maybe just make this the default in the `.get("namespace")` line?

Reply:
Nice